src/installing-workflow/experimental-native-ingress.md

```diff
@@ -0,0 +1,80 @@
+# Experimental Native Ingress
```

Comment:
I don't think it's a trivial concern. We generally accept the practice of avoiding DRY exceptions in code. imo, docs shouldn't be any different. In either case, a DRY exception represents an on-going maintenance burden.

Though I don't think it's a trivial concern, I'm not going to belabor it either. You're doing the legwork on this, so I defer to your judgement.